### PR TITLE
[docs] `sky status --kubernetes` docs

### PR DESCRIPTION
**Pull request by romilbhardwaj**
Follow up to #4040.

Tested:
- [x] Rendered locally

### CODE REVIEW

File: docs/source/reference/kubernetes/kubernetes-setup.rst [hunk 1]
```diff
@@ -274,8 +284,47 @@ All SkyPilot tasks are run in pods inside a Kubernetes cluster. As a cluster adm
 you can inspect running pods (e.g., with :code:`kubectl get pods -n namespace`) to check which
 tasks are running and how many resources they are consuming on the cluster.
 
-Additionally, you can also deploy tools such as the `Kubernetes dashboard <https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/>`_ for easily viewing and managing
-SkyPilot tasks running on your cluster.
+Below, we provide tips on how to monitor SkyPilot resources on your Kubernetes cluster.
+
+.. _kubernetes-observability-skystatus:
+
+SkyPilot CLI
```
Comment (concretevitamin) [inline docs/source/reference/kubernetes/kubernetes-setup.rst:291]: "See SkyPilot resources across all users"?

File: docs/source/reference/kubernetes/kubernetes-setup.rst [hunk 2]
```diff
@@ -274,8 +284,47 @@ All SkyPilot tasks are run in pods inside a Kubernetes cluster. As a cluster adm
...
+^^^^^^^^^^^^
+
+We provide a convenience command, :code:`sky status --kubernetes`, to view the status of all SkyPilot resources in the cluster.
```
Comment (concretevitamin) [inline docs/source/reference/kubernetes/kubernetes-setup.rst:294]: Could we replace --kubernetes with --k8s? People will use whatever the docs recommend.

**Review by concretevitamin — COMMENTED** [review]
Nice! I think this is also very useful for MLEs. Could we copy/paste the snippets to a new section below https://skypilot.readthedocs.io/en/latest/reference/kubernetes/kubernetes-getting-started.html#launching-your-first-task ?

File: docs/source/reference/kubernetes/kubernetes-setup.rst (hunk 1 above)
Comment (romilbhardwaj) [reply docs/source/reference/kubernetes/kubernetes-setup.rst]: Changed to `List SkyPilot resources across all users`

**Comment by romilbhardwaj** [comment]
Thanks, added a section to the getting started page as well.

File: docs/source/reference/kubernetes/kubernetes-getting-started.rst [hunk 3]
```diff
@@ -119,6 +119,57 @@ Once your cluster administrator has :ref:`setup a Kubernetes cluster <kubernetes
     $ kubectl config set-context --current --namespace=mynamespace
 
 
+
+Viewing cluster status
```
Comment (concretevitamin) [inline docs/source/reference/kubernetes/kubernetes-getting-started.rst:123]: Q: thinking behind using 2 titles for the same ~ similar content on 2 pages? Shall we keep this one?

**Pull request merged**